rbac/processor/role/role_members.py

```diff
@@ -103,6 +106,21 @@ def apply_propose_remove(header, payload, state):
         state=state,
     )
 
+def hierachical_approve(header, payload, state):
```

Comment:
"hierarchical"

Reply:
Please update across all instances in all files in this commit. It seems mostly consistent, but I did see at least one example of spelling it appropriately. Bigger picture the question is, whether this should be changed altogether, but lets fix and merge this feature for now.